orderapi/src/main/java/fi/hel/verkkokauppa/order/service/refund/RefundService.java

```diff
@@ -0,0 +1,36 @@
+package fi.hel.verkkokauppa.order.service.refund;
+
+import fi.hel.verkkokauppa.order.model.refund.Refund;
+import fi.hel.verkkokauppa.order.model.refund.RefundAccountingStatus;
+import fi.hel.verkkokauppa.order.repository.jpa.RefundRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDate;
+import java.util.Optional;
+
+
+@Component
+public class RefundService {
+
+    private Logger log = LoggerFactory.getLogger(RefundService.class);
```

Comment:
This logger can be done with class annotation @Slf4j

Reply:
done